CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # v 2.11.0 (?)
```

Comment:
I wonder if we need to make this a 3.0.0 if we drop IO ? 

We could, if we want to, put the import `from inmanta.agent import io as agent_io` in the method body and keep support for iso7 ?

Reply:
only std and exec used it and those modules have been ported and updated.

Depends if we want to keep supporting the iso7 branch of std with the latest pytest-inmanta


I agree with the major bump.

Reply:
Given that the std resources are only used by us internally, I am inclined to just handle it in the std iso7 branch instead of poluting pytest-inmanta even more

Reply:
For the version: isn't that the release tool that bumps this to 3.0.0?

Reply:
good question, I'm not sure